core/tests/protractor_desktop/topicAndStoryEditorFileUploadFeatures.js

```diff
@@ -128,6 +128,43 @@ describe('Topic editor functionality', function() {
     topicEditorPage.expectSubtopicPageContentsToMatch('Subtopic Contents');
   });
 
+  it('should publish and unpublish a story correctly', function() {
```

Comment:
Hi @U8NWXD, I've moved this test from the topicAndStoryEditor suite to here because it includes some file upload features (which works better in Circle CI).